### ISSUE TITLE
Make examples easier to find

### ISSUE THREAD

**Issue by oalders** [metacpan/metacpan-web#2023]
I often forget that a dist comes with some scripts in an examples directory.  I'd like to see one or both of the following:

1) A link to the examples folder in the left sidebar, much like we have a link to the Changes.  We'd need to look in a few places based on common names for folders containing examples.  This would be a shortcut to browsing the dist in that particular folder.
2) Add the example scripts to the release page.  They could be listed out individually and linked.  This could live above the "Other files" section

Basically this is a form of documentation that would could do a better job of highlighting.

**Comment by mickeyn** [comment]
1. I like the idea for the link on the side
2. exists in the release page - see [MetaCPAN::Client](https://metacpan.org/release/MetaCPAN-Client)

**Comment by oalders** [comment]
> Exists in the release page - see MetaCPAN::Client

Wow, I missed that completely.  :)  Maybe the sidebar link would help make it more obvious then.

**Comment by haarg** [comment]
Dupe of #1626
